cli/index.js

```diff
@@ -87,12 +91,22 @@ const tfs = (t = cli.flags.transforms) =>
  */
 if (cli.input[0]) {
   // If there is first argument, then read this file
+  const args = [
+    tfs(),
+    cli.flags.locale,
+    cli.flags.outFile
+  ];
+
   try {
-    render(
-      fs.readFileSync(cwd(cli.input[0])).toString(),
-      tfs(),
-      cli.flags.locale
-    );
+    render(fs.readFileSync(cli.input[0], 'utf8'), ...args);
+    if (cli.flags.watch) {
+      console.log(`Watching ${cli.input[0]}...`);
+      watcher.watch(cli.input[0])
+        .on('change', path => {
+          console.log(`${path} has been changed.`);
+          render(fs.readFileSync(path, 'utf8'), ...args);
+        });
+    }
   } catch (e) {
     console.error(`Something went wrong: ${e.message}`);
```

Comment:
i know, maybe its better to just to `throw err` here? because stacktrace is kinda important for debug